solutions/5.js

```diff
@@ -0,0 +1,8 @@
+const sayHi = (func,num, i=0)=>{
+  if(i==num){
```

Comment:
if(i<num){
func();
sayHi(func, num, i+1);
} 
***This way, there's no need to return anything.

Reply:
makes the code look cleaner...song made this comment on minesss